input/include/NovelRT/Input/KeyState.hpp

```diff
@@ -15,10 +15,10 @@ namespace NovelRT::Input
      */
     enum class KeyState : int32_t
     {
-        Idle = 0,
-        KeyUp = 1,
-        KeyDown = 2,
-        KeyDownHeld = 3
+        Idle = -1,
```

Comment:
Can you clarify this change for me?

Reply:
This change was made to more directly match how GLFW registers changes - mainly since I had found an issue where mouse button clicks were not registering properly based on the previous enum, but switching to this setup resolved it.